src/sounddeviceportaudio.h

```diff
@@ -96,7 +96,7 @@ class SoundDevicePortAudio : public SoundDevice {
     ControlObjectSlave* m_pMasterAudioLatencyOverload;
     int m_underflowUpdateCount;
     static volatile int m_underflowHappend;
-    int m_nsInAudioCb;
+    qint64 m_nsInAudioCb;
```

Comment:
This should be a performance regression on a 32 bit CPU. Is a cast form qint64 to in expensive? We may consider to cast to double because there is a double cast later anyway. Maybe SSE register are used than.